metricbeat/module/kubernetes/pod/_meta/fields.yml

```diff
@@ -78,3 +78,35 @@
               format: percentage
               description: >
                 Memory usage as a percentage of the defined limit for the pod containers (or total node allocatable memory if unlimited)
+        - name: available
+          type: group
+          fields:
+            - name: bytes
+              type: long
+              format: bytes
+              description: >
+                Total memory available
+        - name: workingSet
```

Comment:
working_set, page_faults? We normally don't use camelCase

Reply:
Umm, it was like this in the original PR, I'll open and backport a fix for this.

Reply:
SGTM. Let's get this in and then also do a backport. Probably slipped through in the review on master :-)

Reply:
I have updated this branch with the fix for names so we backport both things at once :)